README.md

```diff
@@ -1,6 +1,10 @@
 # X9Containers
 Docker containers to finger pointing pesky breaches from your running images.
 
+It's nothing really new: X9Containers use existing tools to find malwares and vulnerabilities using [Trivy](https://github.com/aquasecurity/trivy) and [ClamAV](https://github.com/Cisco-Talos/clamav) in a [multi-stage building approach](https://docs.docker.com/develop/develop-images/multistage-build/), copying the whole filesystem to be analyzed at each stage - at `distroless.` suffix.
+
+But the analysis can also occur during building, so it can run into the intermediate container, where we hope to find things dynamically - at non `distroless.` suffix.
+
 X9 it is a popular slang among Brazilians wich can be expressed as whistleblower.
```

Comment:
`wich` => `which means whistleblower`

Reply:
Three witches watch three Swatch watches. Which witch watch which Swatch watch?

Reply:
@carloosamarante the answer is [Wishiwashi (Pokémon)](https://bulbapedia.bulbagarden.net/wiki/Wishiwashi_(Pok%C3%A9mon)#:~:text=Wishiwashi%20(Japanese%3A%20%E3%83%A8%E3%83%AF%E3%82%B7%20Yowashi),it%20has%20reached%20level%2020.).